### ISSUE TITLE
What's up?

### ISSUE THREAD

**Issue by RoscaS** [FabianLauer/chrome-vs-code#3]
Hey,

I was wondering what's going on with your project.

Cheers from Switzerland

**Comment by ScorpioCoding** [comment]
@RoscaS 
Hi there,
I was wondering the same thing.
Cheers from Belgium

**Comment by FabianLauer** [comment]
Hey, nice to see people are interested :slightly_smiling_face: 

I started this a while ago as an experiment. Since the UI is pretty hacky (see [this comment](https://github.com/Microsoft/vscode/issues/15437#issuecomment-260270341)) I'm not sure the extension will ever become fully stable; this is why I didn't continue to work on it.

If you guys are interested in hacking on the project, PRs are always welcome. Maybe I'll find the time to work on it myself again, depending on community interest. Here's an overview on the status quo of the project BTW: https://github.com/FabianLauer/chrome-vs-code#key-features.

So, if you have any questions or features you want to see, feel free to open issues and PRs.

Cheers :v:

**Comment by RoscaS** [comment]
I'd love to but I'm in my first year of CS and I'm pretty sure that my C or Assembly skills wouldn't be really helpful :D.

**Comment by iameronmosele** [comment]
what does it take to contribute on this project as i really need this feature in vs-code asap

**Comment by FabianLauer** [comment]
@iameronmosele Considering that this isn't a feature but an experimental extension of VS Code, may I ask why you need it asap? Just curios.

**Comment by ScorpioCoding** [comment]
With a build in browser within the IDE you hit 'F11' and work in FULL-SCREEN mode.
Thus having your browser a part of your IDE it they render together when in full-screen mode.
The diiference between having a preview and a functioning browser is that from within in full-screen mode you can browser Guides, Resources, CodeTech sites etc.  Also your local server configuration files like PhpMyAdmin
Working in full-screen mode just kills all other distractions. And also in full-screen mode making tutorials, youvideo's etc it's just way easier...
But thats just me...lol

**Comment by FabianLauer** [comment]
@ScorpioCoding Agreed. There's more use cases – imagine working on CSS/Less/SASS with a realtime preview of your changes right next to your editor :heart_eyes: 

But actually I was just wondering why @iameronmosele needs it "asap" – it's not like you can't have a web browser next to the editor.

Regarding what it takes to contribute; as I said earlier, issues and PRs are very welcome :+1:

**Comment by RoscaS** [comment]
@ScorpioCoding that's pretty much why I reckon this feature is a must have. ATM what I'm doing is that I use the full screen mode of VScode and a plugin for Chrome that allows me to pop-upise any tab so I don't have the navigation/toolbar in the way. My linux distro allows me to bind a key to keep a window "allways on top" so altogether this setup kind of mimicking what you're looking for. If you're on Windows I remember that you can use a tool to easily force a window to stay on top but can't remember. the name of it. If you need it, tell me and I'll try to find it on my Windows setup.

Cheers and happy coding everybody :)

**Comment by pavittarx** [comment]
Well, I came to know, its experimental build, but I would love to use/test it, still I am not sure how to install it? 
The documentation doesn't mention Installation process, and the extension isn't available at VSCode market place.

**Comment by FabianLauer** [comment]
There's now a VSIX included in this repo. The extension is not published to the marketplace yet (and I'm not sure it ever will be), so you'll need to [download the latest VSIX](https://github.com/FabianLauer/chrome-vs-code/blob/master/chrome-vs-code-0.0.1.vsix) directly from this repo and then [install from VSIX](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix).

Have fun (it's *very* buggy). Oh, and, PRs welcome 😉

**Comment by thinktopdown** [comment]
@FabianLauer this is awesome! silly question. I installed it via the CLI, but now how to I get to open in a tab in VSCode?

**Comment by FabianLauer** [comment]
@thinktopdown Press <kbd>F1</kbd>, type "Browser", select "Open Web Browser" 🙂

**Comment by VladislavAnkudinov** [comment]
Any alternatives to this extension? I saw bunch of feature requests in vscode repo but all declined though

**Comment by pavittarx** [comment]
@VladislavAnkudinov for the time being it is one of a kind & no extension similar to this exist. VS Code has no intention to provide a built in browser ,so if you want this is your only hope. So don't expect it to be coming into VS Code.

**Comment by TheSETJ** [comment]
This extension + Live server is amazing. I wish I could help developing it. Unfortunately, I'm not experienced enough. :(

**Comment by jkanchelov** [comment]
@FabianLauer Still why not publish the extension even with the experimental tag and the 0.0.1 version? 

I find it handy for my type of work right now and it would be easier to install it to few of my workspaces that way. The plus side is this and that it has a chance to make this repo more popular and bring some people who want and have time to contribute to it.

**Comment by weiminghuaa** [comment]
我是从iOS开发转到前端开发的，最开始我是做java的，我很诧异，为什么前端的开发工具，没有一个集成chrome。难道都是双屏开发吗，1个屏是浏览器，用于调试，1个屏是vscode，用于编辑代码。调试前端代码时，console竟然是到chrome才能看到，而不是vscode，做iOS或后端没有经历这种痛苦，因为eclipse或xcode都自带了程序运行时的console。我想，这个项目真棒。如果大家有兴趣，中国的腾讯公司做了一个微信开发者工具，实现了类似功能，并在中国广泛使用，不过，它仅能开发微信的小程序。

I switched from iOS development to front-end development. At the beginning, I was doing java. I was surprised. Why is the front-end development tool not integrated with chrome? Is it dual-screen development? One screen is a browser for debugging, and one screen is vscode for editing code. When debugging the front-end code, the console is actually visible to chrome, not vscode. The iOS or the backend does not experience this kind of pain, because eclipse or xcode comes with the console at runtime. I think this project is awesome. If everyone is interested, China's Tencent has made a WeChat developer tool that implements similar functions and is widely used in China. However, it can only develop WeChat applets.

**Comment by TEEN-BOOM** [comment]
Could this be published?
